library/Fission/Internal/Orphanage/Signature.hs

```diff
@@ -0,0 +1,18 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+module Fission.Internal.Orphanage.Signature () where
+
+
+import Fission.Prelude
+
+import Servant
+import Servant.Swagger
+import Data.Swagger
+
+import Fission.Web.Auth.Types
+
+-- @@TODO: add an accurate security definition
```

Comment:
The only [SecuritySchemeTypes](https://hackage.haskell.org/package/swagger2-2.5/docs/Data-Swagger-Internal.html#t:SecuritySchemeType) :thinking: 

Could use a hand on this^^

Reply:
Should be something along the lines of:

```haskell
let
  apiKey = ApiKeyParams "Authorization" ApiKeyHeader
in
...
|> securityDefinitions .~ [( "apiKey", SecurityScheme (SecuritySchemeApiKey apiKey) Nothing )]
```

Is this what you could use help with, or was it something else?

Reply:
Yeah our jwt just isn't really an api key. It's more like a transaction signature. But there's no Swagger type for that. ApiKey is probably the closest, so maybe it's best to just use that